packages/opentelemetry-core/src/trace/NoopSpan.ts

```diff
@@ -15,30 +15,18 @@
  */
 
 import * as types from '@opentelemetry/types';
-import { randomSpanId, randomTraceId } from '../common/util/id';
 
-/** This is a base/default implementation of {@link Span}. */
-export class BaseSpan implements types.Span {
-  name = 'DefaultSpan'; // default name
-  spanId = '';
-  traceId = '';
-  parentSpanId = '';
-  kind = types.SpanKind.INTERNAL; // default kind
+const INVALID_SPAN_CONTEXT = { traceId: '', spanId: '' };
```

Comment:
This should probably be a real `SpanContext`. This is especially true if propagation is received and marked as unsampled, in which case it would be more lightweight to rely on a single `NoopSpan` for the trace. However, since W3C still requires to propagate the IDs then it should be a valid `SpanContext`.

Reply:
Makes sense, done. PTAL